lib/iris/analysis/maths.py

```diff
@@ -569,11 +577,11 @@ def _binary_op_common(operation_function, operation_noun, cube, other,
         other = _broadcast_cube_coord_data(cube, other, operation_noun, dim)
     elif isinstance(other, iris.cube.Cube):
         try:
-            np.broadcast_arrays(cube.data, other.data)
+            BA.broadcast_arrays(cube._my_data, other._my_data)
```

Comment:
Yeah I agree. I was more thinking that it might be preferable to use a public function?